test/init_test.go

```diff
@@ -171,5 +171,14 @@ func getCRDYaml(cs *clients) ([]byte, error) {
 	for _, i := range trs.Items {
 		printOrAdd("TaskRun", i.Name, i)
 	}
+
+	pods, err := cs.KubeClient.Kube.CoreV1().Pods(ns).List(metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+	for _, i := range pods.Items {
+		printOrAdd("Pod", i.Name, i)
```

Comment:
so glad that we'll get pod debug output now!! :tada:

Reply:
I needed that info to debug so made sense to include it